CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+If you want to make changes to the schema, start in the `jsonschema` directory where you can edit the JSON files.
+
+Some guidelines for making schema changes:
+
+- Use the most appropriate data type
+- Follow existing naming conventions (camelCase)
+- Add new fields last to minimise impact on consuming code
+- Probably don't add a new field as `required` - this will make the new code unable to read existing messages
```

Comment:
I don't think this is a problem. If a field is missing, what should consumers do when they need that field?